resources/asciidoctor/spec/edit_me_spec.rb

```diff
@@ -89,6 +89,29 @@
     expect(convert input, attributes).to eq(expected.strip)
   end
 
+  it "doesn't add extra path segments if repo_root is the base_dir" do
+    attributes = {
+      'edit_url' => 'www.example.com/docs/',
+      'repo_root' => File.dirname(__FILE__),
+    }
+    input = <<~ASCIIDOC
+      include::resources/edit_me/chapter1.adoc[]
+
+      include::resources/edit_me/chapter2.adoc[]
+    ASCIIDOC
+    expected = <<~DOCBOOK
+      <chapter id="_chapter_1">
+      <title>Chapter 1<ulink role="edit_me" url="www.example.com/docs/resources/edit_me/chapter1.adoc">Edit me</ulink></title>
+      <simpara>Words.</simpara>
+      </chapter>
+      <chapter id="_chapter_2">
+      <title>Chapter 2<ulink role="edit_me" url="www.example.com/docs/resources/edit_me/chapter2.adoc">Edit me</ulink></title>
+      <simpara>Words.</simpara>
+      </chapter>
+    DOCBOOK
+    expect(convert input, attributes).to eq(expected.strip)
+  end
+
```

Comment:
There's a lot of HTML here that doesn't directly support the intent of the test. Could you "zoom in" on the relevant parts with "includes" assertions, rather than a big "is" assertion?

Reply:
I can do that but the *file* is about testing the edit me link itself and its location in the xml is important. So I can't do that for the whole file.